exercises/concept/squeaky-clean/.docs/instructions.md

```diff
@@ -26,11 +26,22 @@ SqueakyClean.clean("a-bc");
 // => "aBc"
 ```
 
-## 3. Omit characters that are not letters
+## 3. Convert leetspeak to normal text
+
+Modify the (_static_) `SqueakyClean.clean()` method to convert [leetspeak][leet-speak] to normal text.
```

Comment:
I think we should clarify in the instructions exactly which numbers they should account for, since the Wiki page contains a _huge_ table.

Reply:
Ah you commented the same thing:

> I added the new task, I'm not sure if I should add a note that says that we are not using all the numbers from leetspeak or if the students should realize this checking the tests.

I think that in general it's fine for _practice_ exercises to require students to look at the tests in order to find out what is required of them. For _concept_ exercises however, I'd like to hold their hands a little more, as it's all about learning a new concept here. So I guess that is the judgement call I would make in most cases.

Reply:
> For concept exercises however, I'd like to hold their hands a little more, as it's all about learning a new concept here. So I guess that is the judgement call I would make in most cases.

Yeah I agree! we go for this one then